backend/datautil/timeutil/timeutils.py

```diff
@@ -142,7 +142,11 @@ def get_times(time_col):
             for hour_time in t_interval.split('-'):
                 # Add 'm' to convert it to Python recognizable format
                 hour_time += 'm'
-                formatted_interval = datetime.strptime(hour_time, '%I:%M%p')
+                try:
+                    formatted_interval = datetime.strptime(hour_time, '%I:%M%p')
+                except ValueError:
+                    print("[timeutils] invalid hour_time of time interval: " + str(hour_time))
```

Comment:
just say invalid hours, remember the variable names can change but what they mean doesn't. For example I may change the name of the variable to hour_str, but it still means hours.